### PR TITLE
Generate button toolbar kit

### PR DESCRIPTION
**Pull request by kellyeryan**
#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUX-1183

#### Github Ticket URL

https://github.com/powerhome/playbook/issues/389

#### Screens

Screen Shot 2020-07-31 at 2 17 28 PM

Screen Shot 2020-07-31 at 2 17 35 PM

Screen Shot 2020-07-31 at 2 15 08 PM

Screen Shot 2020-07-31 at 2 15 24 PM

#### Breaking Changes

I have added a new kit, so this may break.

#### Checklist:

- [X] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [X] **SCREENSHOT** Please add a screen shot or two
- [X] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -223,4 +223,4 @@ DEPENDENCIES
   web-console (= 3.7.0)
 
 BUNDLED WITH
-   1.17.3
+   1.17.3
```
Comment (thestephenmarshall) [inline Gemfile.lock:226]: This file shouldn't be manually edited. There should be a new line at the end of each file.

File: app/pb_kits/playbook/pb_title/title.rb [hunk 2]
```diff
@@ -31,4 +31,4 @@ def dark_class
       end
     end
   end
-end
+end
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_title/title.rb:34]: Add a new line here

File: lib/playbook/version.rb [hunk 3]
```diff
@@ -2,4 +2,4 @@
 
 module Playbook
   VERSION = "6.1.0"
-end
+end
```
Comment (thestephenmarshall) [inline lib/playbook/version.rb:5]: New line here

File: package.json [hunk 4]
```diff
@@ -93,4 +93,4 @@
       "pre-commit": "lint-staged"
     }
   }
-}
+}
```
Comment (thestephenmarshall) [inline package.json:96]: New line

File: spec/pb_kits/playbook/kits/title_spec.rb [hunk 5]
```diff
@@ -14,7 +14,7 @@
     .with_default("h3")
      }
   it { is_expected.to define_enum_prop(:variant)
-     .with_values(nil, "primary")
+     .with_values(nil, "link")
```
Comment (thestephenmarshall) [inline spec/pb_kits/playbook/kits/title_spec.rb:17]: Why is this spec changing in this PR. This seems out of place.

Comment (kellyeryan) [reply spec/pb_kits/playbook/kits/title_spec.rb:17]: This was changed because of the changed to the Title kit made by Jason last Friday.

Comment (thestephenmarshall) [reply spec/pb_kits/playbook/kits/title_spec.rb:17]: Ok. I think you may need to rebase off of `6.1.0` then to make this change disappear from the PR.

**Pull request merged**